youtube_dl/extractor/svt.py

```diff
@@ -44,7 +45,8 @@ def _extract_video(self, video_info, video_id):
                     'format_id': player_type,
                     'url': vurl,
                 })
-        if not formats and video_info.get('rights', {}).get('geoBlockedSweden'):
+        rights = video_info.get('rights', {})
+        if not formats and rights.get('geoBlockedSweden'):
```

Comment:
`rights` is not guaranteed to be a dict.

Reply:
change to use `try_get`